test/module/irohad/torii/query_service_test.cpp

```diff
@@ -164,3 +176,121 @@ TEST_F(QueryServiceTest, InvalidWhenDuplicateHash) {
           shared_model::interface::StatelessFailedErrorResponse>(),
       resp.get()));
 }
+
+class BlocksQueryServiceTest : public ::testing::Test {
```

Comment:
It seems that this test is rather similar to `ToriiQueriesTest`. Consider moving it to `torii_queries_test.cpp` and reusing the fixture, because it contains mostly the same code.